src/theory/quantifiers/ceg_instantiator.cpp

```diff
@@ -114,30 +114,44 @@ bool CegInstantiator::doAddInstantiation( SolvedForm& sf, unsigned i, unsigned e
   if( i==d_vars.size() ){
     //solved for all variables, now construct instantiation
     bool needsPostprocess = false;
-    std::map< Instantiator *, Node > pp_inst;
+    std::vector< Instantiator * > pp_inst;
+    std::map< Instantiator *, Node > pp_inst_to_var;
```

Comment:
Does the order of the elements matter for `pp_inst_to_var`?

Reply:
If not, use unordered_map instead.

Reply:
this is TBD, I was fluctuating on this, I'll address this in my next commit